docs/hooks/useScaffoldEventHistory.md

```diff
@@ -29,4 +29,15 @@ const {
 });
 ```
 
-This example retrieves the historical event logs for the `GreetingChange` event of the `YourContract` smart contract, starting from block number 31231 and filtering events where the premium parameter is true. The data property of the returned object contains an array of event objects, each containing the event parameters and (optionally) the block, transaction, and receipt data. The `isLoading` property indicates whether the event logs are currently being fetched, and the `error` property contains any error that occurred during the fetching process (if applicable). Since `watch` is set to true, the event logs will be refetched every [`pollingInterval`](/deploying/deploy-nextjs-app#--pollinginterval) (set at `scaffold.config.ts`).
+This example retrieves the historical event logs for the `GreetingChange` event of the `YourContract` smart contract, starting from block number 31231 and filtering events where the premium parameter is true.
+
+## Parameters
+
+All the parameters accepted by `useScaffoldEventHistory` hooks are explained in above code block example with comments.
```

Comment:
Lol don't know if this is too informal, I added it at https://github.com/scaffold-eth/se2-docs/pull/40/commits/183e59b5351d30a1a1004193cd9781649a30b30f

Don't know if we should remove comments from code example and explain all the parameters here OR we could completely remove "Parameters" section for these hooks since we already described them at code block example

Or this looks fine?

Reply:
Thanks for the update and feedback @technophile-04 🙌

> Don't know if we should remove comments from code example and explain all the parameters here OR we could completely remove "Parameters" section for these hooks since we already described them at code block example

Reading all hooks again, I would vote to either getting rid of "Parameters" section, or removing the code comments and do a formal explanation of each parameter in the "Parameters" section. I think any of those could be a better implementation than the current "intermediate" one. 

Maybe simpler one can be just remove "Parameters", but we can wait for other opinions 🙌

Reply:
I'd vote for the latter! Same thing we are doing with component props:

https://docs.scaffoldeth.io/components/Address

Reply:
> I'd vote for the latter! Same thing we are doing with component props:

Adding it! I think it will look much better in readability terms and maintains consistency 🙌